docs/sources/mimir/set-up/helm-chart.md

```diff
@@ -0,0 +1,12 @@
+---
+description: Learn how to deploy Mimir with Helm.
+title: Deploy Mimir with Helm
+menuTitle: Deploy with Helm
+weight: 350
+---
+
+# Deploy Mimir with Helm
+
+The [`mimir-distributed` Helm chart](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/README.md) for Grafana Mimir and Grafana Enterprise Metrics allows you to configure, install, and upgrade Grafana Mimir or Grafana Enterprise Metrics within a Kubernetes cluster.
+
+To deploy Mimir using the `mimir-distributed` Helm chart, see [Get started with Grafana Mimir using the Helm chart](/docs/helm-charts/mimir-distributed).
```

Comment:
The get started page appears to be at https://grafana.com/docs/helm-charts/mimir-distributed/latest/get-started-helm-charts/.

```suggestion
To deploy Mimir using the `mimir-distributed` Helm chart, see [Get started with Grafana Mimir using the Helm chart](/docs/helm-charts/mimir-distributed/latest/get-started-helm-charts/).
```